README.md

```diff
@@ -150,54 +166,81 @@ logging_outputs:
 - `logging_enabled` : When 'true', logging role will deploy specified configuration file set. Default to 'true'.
 - `logging_purge_confs`: By default, the Rsyslog configuration files are applied on top of pre-existing configuration files. To purge local files prior to setting new ones, set logging_purge_confs variable to 'true', it will move all Rsyslog configuration files to a backup directory, `/tmp/rsyslog.d-XXXXXX/backup.tgz`, before deploying the new configuration files. Defaults to 'false'.
 - `logging_mmk8s_token`: Path to token for kubernetes.  Default to "/etc/rsyslog.d/mmk8s.token".
 - `logging_mmk8s_ca_cert`: Path to CA cert for kubernetes.  Default to "/etc/rsyslog.d/mmk8s.ca.crt".
-
 - `logging_outputs`: A set of following variables to specify output configurations.  It could be an list if multiple outputs that should to be configured.
-   -  **If `type: elasticsearch`**, send logs to one or more remote elasticsearch or Viaq installations.
-      - `name`: Name of the elasticsearch element.
-      - `type`: Type of the output element. Optional values: `elasticsearch`, `local`, `custom_files`.
-      - `logging_inputs` : List of optional logs collections, dictionaries with `name`, `type` and `state` attributes, that were pre-configured.
-        - `name`: Unique name of the input.
-          `type`: The type of the pre-configured logs to collect. **Note:** Currently ['viaq', 'viaq-k8s', 'ovirt'] are supported for the elasticsearch output.
-          
-          `state`: The state of the configuration files states if they should be `present` or `absent`. Default to `present`.
+   - `name`: Unique name of the output
+   - `type`: Type of the output element. Currently, `elasticsearch`, `files`, and `forwards` are supported. The `type` is used to specify a role in roles/rsyslog/roles/output_roles/.
+   -  ** `type: elasticsearch`**
       - `server_host`: Hostname elasticsearch is running on.
       - `server_port`: Port number elasticsearch is listening to.
       - `index_prefix`: Elasticsearch index prefix the particular log will be indexed to.
+      - `input_type`: Specifying the input type. Type `ovirt` and `viaq` are supported. Default to `ovirt`.
+      - `retryfailures`: Specifying whether retries or not in case of failure. on or off.  Default to on.
       - `ca_cert`: Path to CA cert for ElasticSearch.  Default to '/etc/rsyslog.d/es-ca.crt'
       - `cert`: Path to cert for ElasticSearch.  Default to '/etc/rsyslog.d/es-cert.pem'
       - `key`: Path to key for ElasticSearch.  Default to "/etc/rsyslog.d/es-key.pem"
+   -  ** `type: files`**
+      - `facility`: facility; default to `*`
+      - `severity`: severity; default to `*`
+	  - `exclude`: exclude; default to none.
+	  - `path`: path to the output file.  Must have.  If `path` is not defined, the files instance is dropped.
```

Comment:
indentation - I think `path` is supposed to be at the same level as `severity`

Reply:
For some reason, 'tab' had sneaked into... :(  I'm checking the other files, too.  Thanks!